test/unit/instance-id/instance-id.spec.ts

```diff
@@ -171,10 +173,11 @@ describe('InstanceId', () => {
         });
     });
 
-    it('should throw an error when the backend returns an error', () => {
+    it('should throw a FirebaseInstanceIdError error when the backend returns an error', () => {
```

Comment:
Do backticks work in this kind of context?

Reply:
This is part of the test source. Not related to docs, and doesn't support any formatting.